src/main/scala/io/picnicml/doddlemodel/preprocessing/QuantileDiscretizer.scala

```diff
@@ -0,0 +1,106 @@
+package io.picnicml.doddlemodel.preprocessing
+
+import breeze.linalg.DenseVector
+import breeze.stats.DescriptiveStats
+import cats.syntax.option._
+import io.picnicml.doddlemodel.CrossScalaCompat._
+import io.picnicml.doddlemodel.data.Feature.FeatureIndex
+import io.picnicml.doddlemodel.data.Features
+import io.picnicml.doddlemodel.syntax.OptionSyntax._
+import io.picnicml.doddlemodel.typeclasses.Transformer
+import scala.Double.{MaxValue, MinValue}
+
+case class QuantileDiscretizer(
+  private val bucketCounts: DenseVector[Double],
```

Comment:
If I understood correctly, you implemented this with `IntVector` initially and the tests failed (didn't compile)?

I tried using `IntVector` here and changing the `DenseVector[Double]`s to `DenseVector[Int]`s throughout the code/tests and the tests passed on all three supported versions of Scala for this project (2.11.12, 2.12.9 and 2.13.0). I got some mysterious error once that I can't reproduce anymore, but it went away after deleting the `target/` folder and building again. 

Could you please check again, deleting `target/` if the problem persists? I'm not sure where the problem could be as I'm assuming you are using the dependency versions listed in `project/Dependencies`